publish/android.gradle

```diff
@@ -26,9 +26,15 @@ task kDocJar(type: Jar) {
 }
 
 ext.androidArtifact = { Map config ->
-  return {
-    config = config ?: [:]
+  config = config ?: [:]
+
+  task javaDocReadmeJar(type: Jar) {
+    def readme = project.hasProperty("readme") ? project.readme : "$projectDir/../README.md"
+    from config.get('readme') ?: readme
+    archiveClassifier = "javadoc"
+  }
```

Comment:
We can remove this if  line 25 can use `archiveClassifier = 'javadoc'`.

Reply:
Yeah, but the problem is that not all projects have JavaDocs. For example, the SDK Utils and Manifest Config repos don't have JavaDcos (though maybe they should 🤔 ).

But it is a little strange that Kotlin projects will have jars for both `kdoc` and `javadoc`. So maybe I can change line 25 to `archiveClassifier = 'javadoc'` and then update the if/else below to:

```
    if (javaDocTask) {
      artifact javaDocJar
    } else if (kDocTask) {
      artifact kDocJar
    } else {
      artifact javaDocReadmeJar
    }
```